.mocharc.js

```diff
@@ -3,5 +3,5 @@ const chaiAsPromised = require("chai-as-promised");
 
 chai.use(chaiAsPromised);
 module.exports = {
-    
+  spec: "./tests/**/*.spec.ts",
```

Comment:
the `ts-node` is no longer required?

Reply:
We didn't have `ts-node`.
.mocharc.yaml was some try to fix the issue.
Before it, we had .mocharc.js, but we missed this spec configuration.